forum/users/models.py

```diff
@@ -1,3 +1,26 @@
+from django.core.validators import EmailValidator
 from django.db import models
+import uuid
 
 # Create your models here.
+class Role(models.Model):
+    role_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    role_name = models.CharField(max_length=50, null=False, unique=True)
+
+class User(models.Model):
+    user_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
Consider adding `editable=False` to ensure that users don’t attempt to change the UUID.
_example in line's 7 comment_

Reply:
Added editable=False for user_id field.